docs/networks/node-ops/access-onchain-data/access-nodes/accessing-data/access-api.md

```diff
@@ -1117,3 +1117,37 @@ message EventFilter {
 | event_type | A list of full event types to include. <br/> Event types have 2 formats:<br/> _ Protocol events: `flow.[event name]`<br/> _ Smart contract events: `A.[contract address].[contract name].[event name]`                                                                                                |
 | contract   | A list of contracts who's events should be included. Contracts have the following name formats:<br/> _ Protocol events: `flow`<br/> _ Smart contract events: `A.[contract address].[contract name]`<br/> This filter matches on the full contract including its address, not just the contract's name |
 | address    | A list of addresses who's events should be included. Addresses must be Flow account addresses in hex format and valid for the network the node is connected to. i.e. only a mainnet address is valid for a mainnet node. Addresses may optionally include the `0x` prefix                             |
+
+## Execution Data Streaming API
```

Comment:
@vishalchangrani these are CLI flags the operator should use to configure the way the streaming API works, not settings a user would need to interact with it. Does this belong here?